BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Internal/Tracer.cs

```diff
@@ -105,11 +98,13 @@ private IEnumerator Worker()
 
         public void OnSpanEnd(Span span)
         {
+            var weakSpan = new WeakReference<Span>(span);
```

Comment:
This is only used inside of `if (!_started)`, so no need to instantiate it at the top scope.